fast_api/routes/organization.py

```diff
@@ -18,6 +27,7 @@
 from fast_api.routes.client_response import pack_json_result
 from submodules.model import events
 from submodules.model.business_objects import organization
+from submodules.model.business_objects.user import get
```

Comment:
without as <xx> or indicator a "get" is hard to identify where it's coming from. Usually we import the full object (also to prevent parial import issues) so
```suggestion
from submodules.model.business_objects import user as user_db_bo
```

so we can go with `user_db_bo.get(xxx)` and already see whats happening. 

For some clearly described functions we also work with function imports but since all db objects have a get not optimal here